fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -91,12 +91,17 @@ public static class BaseTableInfo {
         @SerializedName(value = "tableName")
         private String tableName;
 
-        public BaseTableInfo(long dbId, long tableId) {
+        public BaseTableInfo(long dbId, String dbName, long tableId) {
```

Comment:
why add dbName?

Reply:
At replaceTableAlias function, we need baseinfo db name to fill slof tblName